src/plugins/home/server/services/sample_data/lib/insert_data_into_index.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { IScopedClusterClient, Logger } from 'kibana/server';
+import type { DataIndexSchema } from './sample_dataset_registry_types';
+import {
+  translateTimeRelativeToDifference,
+  translateTimeRelativeToWeek,
+} from './translate_timestamp';
+import { loadData } from './load_data';
+
+export const insertDataIntoIndex = ({
```

Comment:
This was extracted from `src/plugins/home/server/services/sample_data/routes/install.ts`